hawc/templates/base.html

```diff
@@ -45,8 +45,14 @@
     {% block extrastyle %}{% endblock %}
 
   </head>
-  <body class="html">
 
+  <body class="html">
+    <script src="https://unpkg.com/htmx.org@1.6.1"></script>
```

Comment:
can you add this to our static files to instead of using a cdn. see the naming and version pattern for other js